### PR TITLE
Change version references to :latest tag

### PR DESCRIPTION
**Pull request by gorshunovr**
Changed version references to :latest tag and linked containers repository for details on software installed into the containers used for documentation builds.

### CODE REVIEW

File: docs/builds.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ if you provide a good reason your documentation needs more resources.
 You can see the current Docker build images that we use in our `docker repository <https://github.com/rtfd/readthedocs-docker-images>`_.
 `Docker Hub <https://hub.docker.com/r/readthedocs/build/>`_ also shows the latest set of images that have been built.
 
-Currently in production we're using the ``readthedocs/build:2.0`` docker image as our default image.
+Currently in production we're using the ``readthedocs/build:stable`` docker image as our default image.
```
Comment (stsewd) [inline docs/builds.rst]: We are using `latest` now.

3.0 is stable and 4.0 is latest

File: docs/builds.rst [hunk 2]
```diff
@@ -133,57 +133,7 @@ but you can change that using a :doc:`config-file/index`.
...
-  * ``libgraphviz-dev``
-  * ``libjpeg``
-  * ``libxml2-dev``
-  * ``libxslt1-dev``
-  * ``pandoc``
-  * ``plantuml``
-  * ``textlive-full``
-
-`More details <https://github.com/rtfd/readthedocs-docker-images/blob/releases/3.x/Dockerfile>`__
+More details on software installed in images could be found in `rtfd/readthedocs-docker-images <https://github.com/rtfd/readthedocs-docker-images>`__ repository by browsing specific tag, marked in README as ``stable``.
```
Comment (stsewd) [inline docs/builds.rst]: We didn't find a way to automate this, but is also a hard to read the docker file to see all the libs that are installed, maybe we should update to match the libs from latest (4.0) and stable (3.0)

File: docs/builds.rst (hunk 1 above)
Comment (gorshunovr) [reply docs/builds.rst]: Done, thank you.

File: docs/builds.rst (hunk 2 above)
Comment (gorshunovr) [reply docs/builds.rst]: It's harder to maintain a list by copy-pasting it on every release. Dockerfile is pretty self-explanatory and easy to read: you even have one lib per line. I would leave this reference to the repository.

**Comment by gorshunovr** [comment]
PR title has been changed to adjust to the updated label.

**Comment by ericholscher** [comment]
Thanks!

**Pull request merged**